ui/js/constants.js

```diff
@@ -220,6 +220,61 @@ export const thPinboardCountError = "Max pinboard size of 500 reached.";
 
 export const thPerformanceBranches = ["autoland", "mozilla-inbound"];
 
+/**
+ * The set of custom Treeherder events.
+ */
+export const thEvents = {
```

Comment:
I wonder if the events constants should be moved to their own file in the future?